aws/resource_aws_egress_only_internet_gateway_test.go

```diff
@@ -85,6 +85,37 @@ func TestAccAWSEgressOnlyInternetGateway_basic(t *testing.T) {
 	})
 }
 
+func TestAccCheckAWSEgressOnlyInternetGateway_tags(t *testing.T) {
```

Comment:
@stefansundin Could you please rename to `TestAccAWSEgressOnlyInternetGateway_tags` so that

```console
make testacc TEST=./aws TESTARGS='-run=TestAccAWSEgressOnlyInternetGateway_'
```

runs all the relevant acceptance tests.
Thanks.

Reply:
Done. I think I copied it from some other resource. Can't remember which one.